coderd/insights.go

```diff
@@ -550,6 +550,9 @@ func parseInsightsStartAndEndTime(ctx context.Context, rw http.ResponseWriter, s
 		{"end_time", endTimeString, &endTime},
 	} {
 		t, err := time.Parse(insightsTimeLayout, qp.value)
+		// strip monotonic clock reading
+		// so presentation time and arithmetic operations are consistent
+		t = t.Round(0)
```

Comment:
I don't think we need to strip the monotonic clock here since we're parsing a string. Monotonic clock should only be present when `time.Now()` is used. 🤔

Reply:
Yeah I just learned that after making this :P